### DISCUSSION TITLE
[Feat Request] Date-time picker component

### DISCUSSION THREAD

**Discussion by 8bitzz** [tailwindlabs/headlessui#711] [Ideas]
### Description:
- I am searching a date-time picker component but haven't found any which have both date picker + time picker with nice design
- It would be great if there is a `date-time picker` component in `headless-ui`

### Design
- I am imagining something similar to this, but have an option to add a `time-picker` 
<img width="303" alt="Screen Shot 2021-08-01 at 09 33 12" src="https://user-images.githubusercontent.com/23531403/127757253-82810d16-bda5-434b-b8e1-4f8fdc05ef90.png">

**Comment by AlexMachin1997** [comment]
I think this would be really cool to add to the core headlessui library.
